datasets/brazil-data-cubes.yaml

```diff
@@ -40,7 +40,7 @@ DataAtWork:
       AuthorName: Brazil Data Cube
       AuthorURL: https://brazil-data-cube.github.io/
     - Title: Tile Map Service to view Image Collections from Brazil Data Cube Catalog
-      URL: https://github.com/brazil-data-cube/bdc-tiler
+      URL: https://github.com/brazil-data-cube/code-gallery/blob/master/jupyter/Python/tiler/bdc-tiler_introduction.ipynb
```

Comment:
@raphaelrpl can you confirm that this url is a valid replacement of the original example that is currently not available?

Reply:
@jeffejefe,

yes, this is a valid url. We are going to update a few steps inside this notebook in order to keep it updated with AWS.
Thanks :D

Reply:
Thanks @raphaelrpl I interpreted that as the suggested replacement url was valid and not the original (e.g. https://github.com/brazil-data-cube/bdc-tiler). Feel free to update as needed!  Happy holidays!